messenger.go

```diff
@@ -330,6 +330,15 @@ func (m *Messenger) Send(to Recipient, message string) error {
 	return m.SendWithReplies(to, message, nil)
 }
 
+//SendGeneralMessage will send the GenericTemplate message
```

Comment:
Would you mind putting a space here, between the `//` and `Send...`, want to make sure that we're following good [GoDoc comment styling](https://blog.golang.org/godoc-documenting-go-code).